libcob/ChangeLog

```diff
@@ -3564,6 +3590,12 @@
 	* INDEXED file test case added and uses a file to synchronize tests
 	* RELATIVE file test case added and uses a file to synchronize tests
 
+2016-12-30  Ron Norman <rjn@inglenet.com>
```

Comment:
that should be included in the data above and the two "test case" entries be moved to tests/Changelog

Reply:
Done.